spec/factories/custom_field_factory.rb

```diff
@@ -28,7 +28,18 @@
 
 FactoryBot.define do
   factory :custom_field do
-    name { 'Custom Field' }
+    transient do
+      _format_name do
```

Comment:
🙋Honest question: Is the `_` prefix a naming convention for transient attributes to indicate they're not assigned on the instance as attributes?

Reply:
Not really, it's more to indicate that it is internal to the factory and it's not meant to be used externally. I did not find a more elegant way to have the `name` attribute customized when using traits.

Contrary to the `default_option` in the `list` trait which is meant to be used when calling the factory. No `_` here.